README.md

```diff
@@ -132,6 +132,10 @@ An array of paths to [CLDR JSON](https://github.com/dojo/i18n#loading-cldr-data)
 }
 ```
 
+#### `copyPublicAssets`: boolean
```

Comment:
Do we really need a flag for this given we can just presume if they have/haven’t a public folder they’re opting in/out?

Reply:
On second thought, I suppose we do not.